events/bingo/entityScripts/bingoWheel/bingoWheel.js

```diff
@@ -12,225 +12,30 @@
-            }
-            injector = Audio.playSound(sound, {
-                position: position,
-                volume: volume
-            });
-        }
-    }
-
-    // *************************************
-    // END UTILITY FUNCTIONS
-    // *************************************
 
     var Wheel = function() {
         _this = this;
     };
 
     Wheel.prototype = {
-        remotelyCallable: ['alreadyCalledNumbersReply'],
-        
-        /* ON LOADING THE APP: Save a reference to this entity ID and its position */
+        /* ON LOADING THE APP: Save a reference to this entity ID */
         preload: function(entityID) {
             _this.entityID = entityID;
         },
 
-        /* ON MOUSE CLICKING THE WHEEL (THIS WILL INCLUDE TRIGGERING ON WHEEL): If a right mouse click, ignore. Otherwise, 
-        if at least 4 seconds have passed since the last spin, check the user's name aginst those allowed to spin the wheel. 
-        If the user is allowed to spin, give the wheel an angular velocity of -10 in the z direction and play a ticking 
-        sound, then set a timeout for 100MS to ensure we do not check velocity before the wheeel is spinning. Next, clear 
-        any interval to be sure we do not have more than one interval running and set a new interval to check the velocity 
-        of the wheel every 100MS. At this point, we have a shuffled list of possible bingo calls and the last number in 
-        that list will be the final number that is chosen. To give the appearance of the wheel spinning through the list, 
-        we update the text entity every interval, iterating over the list of possible calls. When the wheel has slowed to 
-        a minimum velocity, we update the text entity with the final number. so, for every 100MS, we check the velocity. 
-        If velocity is greater than the minimum, less than 0, and we are not on the final interval where the final number 
-        will be shown, we edit the text with the next number in the list of possible numbers and increase the minimum 
-        velocity to dynamically narrow the amount of time before the text entity will change again. If the angular velocity 
-        is between 0 and -0.1 and we are not on the final text edit, the wheel will stop spinning soon so we pop the array 
-        of possible calls and edit the text with the popped call. We send the called number to the server script and set 
-        a flag that this is the final number. Now, the wheel text will not be edited during subsequent intervals and when 
-        it slows to less than -0.05, we consider the spin finished and play the final beep sound, clear the interval, reset 
-        the list of possible bingo calls, and set a timeout for 4seconds before the wheel can be spun again to allow time 
-        for the server script to complete its tasks for this spin. */
+        // When left-clicking/triggering on the wheel, tell the server script on the wheel
```

Comment:
I like how this was shortened to just a 2 line comment

Reply:
samesies 😅